core/server/common/src/main/java/alluxio/underfs/AbstractUfsManager.java

```diff
@@ -92,7 +93,7 @@ public String toString() {
    * Maps from mount id to {@link UfsInfo} instances. This map helps efficiently retrieve
    * existing UFS info given its mount id.
    */
-  private final ConcurrentHashMap<Long, UfsInfo> mMountIdToUfsMap =
+  private final ConcurrentHashMap<Long, UfsInfo> mMountIdToUfsInfoMap =
```

Comment:
Just curious, any reason for using `ConcurrentHashMap` as opposed to `ConcurrentHashMapV8`?